logistics.md

```diff
@@ -203,14 +270,371 @@ these steps:
 Open a terminal and type `echo $SHELL`. If it reads `/bin/bash` then you are all set! If it does not,
 then for all parts of the subsequent instructions, whenever the instructions read “open a terminal,”
 please assume you are to open a terminal, type bash, and the proceed with the instructions as specified.
+
+###### **Git**
+
+You most likely already have Git installed.
+To check if you do:
+- On Debian -  type `sudo apt-get install git` inside the terminal
+- On Fedora - type `sudo yum install git` inside the terminal.
+
+###### **VSCode**
+Go to [visual studio code website](https://code.visualstudio.com/) and click the download button, then run the `.exe` file.
```

Comment:
I found this on there https://school.brainhackmtl.org/setup/

I think that unless you use Wine you can't run a `.exe` on thelinux, right?

```suggestion
    Go to https://code.visualstudio.com/ and click the download button for either the .deb (Ubuntu, Debian) or the .rpm (Fedora, CentOS) file.
    Double-click the downloaded file to install VSCode. (You may be prompted to type your administrator password during the install).
```